YChartsLib/src/main/java/co/yml/charts/ui/piechart/charts/DonutPieChart.kt

```diff
@@ -135,14 +145,23 @@ fun DonutPieChart(
                             )
                             progressSize.forEachIndexed { index, item ->
                                 if (clickedAngle <= item) {
-                                    if (activePie != index)
-                                        activePie = index
+                                    activePie = if (activePie != index)
+                                        index
+                                    else
+                                        -1
```

Comment:
@alexandre-thauvin Could you move this magic number to constant variable with meaning full name.